modules/storage-aws-s3/s3/s3.go

```diff
@@ -178,6 +178,11 @@ func (s *s3) SetMetaStatus(ctx context.Context, className, snapshotID, status st
 	return nil
 }
 
+func (s *s3) InitSnapshot(ctx context.Context, className, snapshotID string) (*snapshots.Snapshot, error) {
+	// TODO: implement
```

Comment:
This should be done in a separate PR, yes?

Reply:
Yes correct :)